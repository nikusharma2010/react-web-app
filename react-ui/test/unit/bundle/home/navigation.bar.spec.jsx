import React from "react";
import { create } from "react-test-renderer";
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NavigationBar from '../../../../src/bundle/home/navigation';
Enzyme.configure({ adapter: new Adapter() });

describe("NavigationBar Component", () => {
    test("it matches the snapshot", () => {
        const component = create(<NavigationBar />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
