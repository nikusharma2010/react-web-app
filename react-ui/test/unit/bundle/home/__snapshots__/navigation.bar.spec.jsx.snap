// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar Component it matches the snapshot 1`] = `
<div
  className="styles__Row-sc-18xwbcb-3 cYIdhM"
>
  <div
    className="styles__Col-sc-18xwbcb-2 jTVgQF"
  >
    <a
      href="/add"
    >
      <div
        className="styles__Button-sc-18xwbcb-6 styles__ButtonPrimary-sc-18xwbcb-7 fKWqEi"
      >
        Add User
      </div>
    </a>
  </div>
  <div
    className="styles__Col-sc-18xwbcb-2 eBtAHL"
  >
    <a
      href="/Search"
    >
      <div
        className="styles__Button-sc-18xwbcb-6 styles__ButtonInfo-sc-18xwbcb-9 hUVWEa"
      >
        Search User
      </div>
    </a>
  </div>
  <div
    className="styles__Col-sc-18xwbcb-2 gAVkdx"
  >
    <a
      href="/list"
    >
      <div
        className="styles__Button-sc-18xwbcb-6 styles__ButtonSuccess-sc-18xwbcb-8 dOgvkn"
      >
        Users
      </div>
    </a>
  </div>
</div>
`;
